Description: Drop rtslib-fb from requirements
 Ubuntu and Debian don't ship this fork of rtslib.
Author: Unknown
Forwarded: not-needed

diff --git a/requirements.txt b/requirements.txt
index 81c6253..7fc9cbb 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -29,7 +29,6 @@ python-swiftclient>=2.2.0
 requests>=1.2.1,!=2.4.0
 Routes>=1.12.3,!=2.0
 taskflow>=0.4
-rtslib-fb>=2.1.39
 six>=1.7.0
 SQLAlchemy>=0.8.4,<=0.8.99,>=0.9.7,<=0.9.99
 sqlalchemy-migrate>=0.9.1,!=0.9.2

